build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+# Build script for the Cryptomator appimage
+
+VERSION=$1
+BUILD_NUMBER=$2
+
+# Create build dir
+rm --force --recursive ./build
+mkdir build
+cd build
+
+# Download buildkit
+curl -L https://github.com/cryptomator/cryptomator/releases/download/${VERSION}/buildkit-linux.zip -o buildkit.zip
+
+# Unzip buildkit
+unzip buildkit.zip
+
+# Store upstream version in variable
+UPSTREAM_VERSION=`cat ./libs/version.txt`
+
+# Create runtime image
+${JAVA_HOME}/bin/jlink \
+    --verbose \
+    --output runtimeImage \
+    --module-path "${JAVA_HOME/}/jmods" \
+    --add-modules java.base,java.logging,java.xml,java.sql,java.management,java.security.sasl,java.naming,java.datatransfer,java.security.jgss,java.rmi,java.scripting,java.prefs,java.desktop,jdk.security.auth,jdk.unsupported,java.net.http,jdk.crypto.ec \
```

Comment:
Just a note: I've been thinking a lot about whether we can determine these modules somehow using `jdeps`

Reply:
Running

```
ls -1 *.jar | xargs jdeps -s --multi-release 14 | awk -F ' -> ' '{print $2}' | grep '^\(java\|jdk\)\.' | sort -u
```

inside of the jar dir gives us an incomplete list (e.g. it is missing `jdk.crypto.ec`). Not sure where it fails to resolve dependencies.

Reply:
Won't be part of this PR.